src/styles/variables.css

```diff
@@ -0,0 +1,10 @@
+:root {
+  --card-background-color: #fff;
+  --card-border-color: #f3f3f3;
+  --card-color-light: #616070;
+  --card-color-dark: #060b35;
+  --card-button-background-color: #00acdc;
+  --card-border-hover-color: #00acdc;
+  --card-button-text-color: #fff;
+  --card-button-text-color-hover: #00acdc;
```

Comment:
You don't need 3 different variables with the same argument, just use 1 of them.

Reply:
Or rename it to just color-accent-blue and color-light

Reply:
i already did it